js/ui/drop_down_button.js

```diff
@@ -330,17 +345,15 @@ let DropDownButton = Widget.inherit({
                     y: -1
                 }
             },
-            contentTemplate: (content) => {
-                const $content = $(content);
-                $content.addClass(DROP_DOWN_BUTTON_CONTENT);
+            contentTemplate: () => {
                 this._list = this._createComponent($("<div>"), List, this._listOptions());
```

Comment:
```suggestion
                const $listContainer = $("<div").appendTo($container);
                this._list = this._createComponent($listContainer, List, this._listOptions());
```

Avoid issues related with render a widget on container that isn't attached to DOM.